reconcile/utils/mr/promote_qontract.py

```diff
@@ -184,6 +184,15 @@ def process(self, gitlab_cli: GitLabApi) -> None:
             replace_text=self.commit_sha,
         )
 
+        # data/services/app-interface/terraform-repo/cicd/ci-int/saas-terraform-repo.yaml
+        self._process_by(
+            "json_path",
+            gitlab_cli=gitlab_cli,
+            path="data/services/app-interface/terraform-repo/cicd/ci-int/saas-terraform-repo.yaml",
+            search_text="$.resourceTemplates[?(@.url == 'https://gitlab.cee.redhat.com/app-sre/terraform-repo-tekton')].targets[?(@.name == 'tf-repo-prod')].parameters.QR_IMAGE_TAG",
```

Comment:
there are 2 matches for url `https://gitlab.cee.redhat.com/app-sre/terraform-repo-tekton`, would it be cleaner to use `name` field `terraform-repo`?

Reply:
Nice find! Fixed.